src/core/core/comment.js

```diff
@@ -0,0 +1,36 @@
+import { commentModel, userModel } from "../models";
+
+const create = async (newComment) => {
+  return await commentModel.create(newComment);
+};
+
+const update = async (body, id) => {
+  return await commentModel.update(body, { where: { id: id } });
+};
+
+const findOne = async (id) => {
+  return await commentModel.findByPk(id);
+};
+
+const findMany = async (postId) => {
+  return await commentModel.findAll({
```

Comment:
make it dynamic just take the query as props.
like -  comment.findMany({id:id})

const findMany = async (query) => {
  return await commentModel.findAll(query)